src/app/app.module.ts

```diff
@@ -1,16 +1,32 @@
-import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-
+import { NgModule } from '@angular/core';
 import { AppComponent } from './app.component';
+import { HttpClient, HttpClientModule } from '@angular/common/http';
+import { TranslateModule, TranslateLoader } from '@ngx-translate/core';
+import { TranslateHttpLoader } from '@ngx-translate/http-loader';
```

Comment:
please organise imports in 3 block separated by an empty line, like this:

import angular essentials (from '@angular....')

import 3rd party libraries (example '@ngx-translate/core';)

import application stuff (components, services, pipes...)

Reply:
I will fix it and you can see what I mean